app/controllers/verify_controller.rb

```diff
@@ -14,4 +15,22 @@ def index
   def retry
     flash.now[:error] = I18n.t('idv.errors.fail')
   end
+
+  def activated
+    redirect_to verify_url unless active_profile?
+  end
+
+  def fail
+    redirect_to verify_url unless idv_attempter.exceeded?
+  end
+
+  private
+
+  def confirm_no_active_profile
```

Comment:
There is already a method for this in the IdvSession concern: `confirm_idv_needed`